src/components/standalone/hotspot/SettingsContent.vue

```diff
@@ -0,0 +1,695 @@
+  getAxiosErrorMessage,
+  NeButton,
+  NeCombobox,
+  NeInlineNotification,
+  NeModal,
+  NeSkeleton,
+  NeTextInput,
+  NeTooltip
+} from '@nethserver/vue-tailwind-lib'
+import { faSave, faRightToBracket, faCircleCheck } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import FormLayout from '@/components/standalone/FormLayout.vue'
+import { useI18n } from 'vue-i18n'
+import { validateIp4Cidr, validateIp4Address, validateRequired } from '@/lib/validation'
+import { ubusCall } from '@/lib/standalone/ubus'
+import { AxiosError } from 'axios'
+import { useUciPendingChangesStore } from '@/stores/standalone/uciPendingChanges'
+
+const { t } = useI18n()
+const uciPendingChangesStore = useUciPendingChangesStore()
+
+interface Login {
+  hostname: string
+  username: string
+  password: string
+}
+
+const loginForm = ref<Login>({
+  hostname: 'my.nethspot.com',
+  username: '',
+  password: ''
+})
+
+interface Configuration {
+  parentHotspot: string
+  unitName: string
+  unitDescription: string
+  networkDevice: string
+  networkAddress: string
+  dhcpRangeStart: string
+  dhcpRangeEnd: string
+  maxClientsAllowed: string
+}
+
+interface ParentHotspot {
+  id: string
+  label: string
+  name: string
+}
+
+interface NetworkDevice {
+  id: string
+  name: string
+  label: string
+}
+
+const configurationForm = ref<Configuration>({
+  parentHotspot: '',
+  unitName: '',
+  unitDescription: '',
+  networkDevice: '',
+  networkAddress: '',
+  dhcpRangeStart: '',
+  dhcpRangeEnd: '',
+  maxClientsAllowed: ''
+})
+
+let isLoggedIn = ref(false)
+let activeConfiguration = ref(false)
+let loading = ref(false)
+let logging = ref(false)
+let loadingUnregister = ref(false)
+let loadingDhcpRange = ref(false)
+let showUnregisterModal = ref(false)
+let emptyDevices = ref(false)
+let saving = ref(false)
+let successSaving = ref(false)
+let hostnameRef = ref()
+let usernameRef = ref()
+let passwordRef = ref()
+let unitNameRef = ref()
+let unitDescriptionRef = ref()
+let networkAddressRef = ref()
+let dhcpRangeStartRef = ref()
+let dhcpRangeEndRef = ref()
+let parentHotspotList = ref<Array<ParentHotspot>>()
+let networkDeviceList = ref<Array<NetworkDevice>>()
+
+let objError = {
+  notificationTitle: '',
+  notificationDescription: '',
+  hostname: '',
+  username: '',
+  password: '',
+  unitName: '',
+  unitDescription: '',
+  networkAddress: '',
+  dhcpRangeStart: '',
+  dhcpRangeEnd: '',
+  maxClientsAllowed: ''
+}
+let error = ref({ ...objError })
+let errorLoadingData = ref({ ...objError })
+let errorSave = ref({ ...objError })
+let errorUnregister = ref({ ...objError })
+let errorDhcpRange = ref({ ...objError })
+
+onMounted(() => {
+  getFirewallData()
+  getConfiguration()
+})
+
+async function getFirewallData() {
+  loading.value = true
+  errorLoadingData.value = { ...objError }
+
+  // Retrieve parent hotspot
+  try {
+    let getParentHotspot = await ubusCall('ns.dedalo', 'list-parents', {})
```

Comment:
`getParentHotspot` sounds like the name of a function, let's rename this variable to something like `listParentsResponse` or simply `res`